### PR TITLE
Update rapidjson

### PR DESCRIPTION
**Pull request by bkotzz**
Note: This is waiting on https://github.com/hunter-packages/rapidjson/pull/2 to go in and a release to be cut before I can update this change. But I made a release on that branch from my fork, and ran it through CI (with merging in the `pkg.rapidjson` from `ingenue/hunter`). I'll update this pull to the `hunter-packages` release once ready.

* I've followed [this guide](https://docs.hunter.sh/en/latest/creating-new/update.html)
  step by step carefully. **[Yes]**

* I've tested this package remotely and have excluded all broken builds.
  Here is the links to the Travis/AppVeyor with status "All passed":

  * https://ci.appveyor.com/project/bkotzz/hunter/builds/20855244
  * https://travis-ci.com/bkotzz/hunter/builds/94044477 (Passes on all builds except the iOS one, which doesn't exist in the latest template https://github.com/ingenue/hunter/blob/pkg.template/.travis.yml)

---

### CODE REVIEW

File: cmake/projects/RapidJSON/hunter.cmake [hunk 1]
```diff
@@ -9,6 +9,17 @@ include(hunter_cmake_args)
 include(hunter_download)
 include(hunter_pick_scheme)
 
+hunter_add_version(
+    PACKAGE_NAME
+    RapidJSON
+    VERSION
+    "1.1.0-p1"
+    URL
+    "https://github.com/bkotzz/rapidjson/archive/v1.1.0-66eb606-p1.tar.gz"
```
Comment (ruslo) [inline cmake/projects/RapidJSON/hunter.cmake]: Released: https://github.com/hunter-packages/rapidjson/releases/tag/v1.1.0-66eb606-p0

File: cmake/projects/RapidJSON/hunter.cmake [hunk 2]
```diff
@@ -9,6 +9,17 @@ include(hunter_cmake_args)
 include(hunter_download)
 include(hunter_pick_scheme)
 
+hunter_add_version(
+    PACKAGE_NAME
+    RapidJSON
+    VERSION
+    "1.1.0-p1"
```
Comment (ruslo) [inline cmake/projects/RapidJSON/hunter.cmake]: Please use `1.1.0-66eb606-p0`

File: cmake/projects/RapidJSON/hunter.cmake [hunk 3]
```diff
@@ -9,6 +9,17 @@ include(hunter_cmake_args)
...
 
+hunter_add_version(
+    PACKAGE_NAME
+    RapidJSON
+    VERSION
+    "1.1.0-66eb606-p0"
+    URL
+    "https://github.com/hunter-packages/rapidjson/archive/v1.1.0-66eb606-p0.tar.gz"
+    SHA1
+    947c0196612c9be6c39c496f84b03234495af5b1
```
Comment (ruslo) [inline cmake/projects/RapidJSON/hunter.cmake]: Should be
```
SHA1
6889242b198cbff7cd8610f214b4218436a4a7a9
```
intead of
```
SHA1
947c0196612c9be6c39c496f84b03234495af5b1
```

Comment (bkotzz) [reply cmake/projects/RapidJSON/hunter.cmake]: My mistake - I updated the URL but forgot the SHA. Will do now

Comment (bkotzz) [reply cmake/projects/RapidJSON/hunter.cmake]:
```
$ openssl sha1 v1.1.0-66eb606-p0.tar.gz 
SHA1(v1.1.0-66eb606-p0.tar.gz)= 6889242b198cbff7cd8610f214b4218436a4a7a9
```

**Comment by ruslo** [comment]
Testing:
* https://ci.appveyor.com/project/ingenue/hunter/builds/20911501
* https://travis-ci.org/ingenue/hunter/builds/466232760?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Uploading:
* https://ci.appveyor.com/project/ingenue/hunter/builds/20912232
* https://travis-ci.org/ingenue/hunter/builds/466245308?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Released: https://github.com/ruslo/hunter/releases/tag/v0.23.69

**Pull request merged**